docker/down.sh

```diff
@@ -2,8 +2,9 @@
 
 ROOT="$(dirname $( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P ))"
 $ROOT/docker/env.sh
+. $ROOT/docker/base.sh
```

Comment:
it is needed to run with `.`? (same as `source`, right?)

Reply:
Yes :) https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-_002e